src/Form/PriceImport/PriceImportByProductListType.php

```diff
@@ -39,5 +39,58 @@ static function (string $productsAsString) {
                 return explode(',', preg_replace('/\s+/', '', $productsAsString));
             }
         ));
+
+        $builder->addModelTransformer(new CallbackTransformer(
```

Comment:
I would do it with a data-mapper. This way it can be externalized easier (and maybe tested) and it is easier to extend (by just decorating it).